Dockerfile

```diff
@@ -28,18 +29,18 @@ RUN service postfix start
 RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
 RUN locale-gen
 
-RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
-RUN curl https://packages.microsoft.com/config/debian/9/prod.list > /etc/apt/sources.list.d/mssql-release.list
+RUN curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /usr/share/keyrings/microsoft-prod.gpg
+RUN curl https://packages.microsoft.com/config/debian/12/prod.list | tee /etc/apt/sources.list.d/mssql-release.list
```

Comment:
I'd rather not send anything to `stdout` during this build if we can help it.
Is there a reason to use `tee` here rather than `>` ?
It looks like it might be from debugging. Please take a look.

Reply:
Honestly? I don't know 😬  I looked at the documentation and that was the suggestion, but I don't have a real reason to use this particular command

Reply:
The difference between `>` and `| tee` is that `| tee` also sends the output to the stdout so you can follow along. I don't think we will need (or want) that when we're building the image. But I wanted to be sure we weren't expected to interpret some special significance to that `prod.list` file.
(It sounds like, no, so please use `>` if you don't have strong feelings otherwise 🙏 )